tests/interface_call_forwarder/interface_forwarder.test.ts

```diff
@@ -0,0 +1,197 @@
+import { expect } from 'chai';
+import { readFileSync, writeFileSync } from 'fs';
+import { describe, it } from 'mocha';
+import { gen_interface, starknetCompile, transpile } from '../util';
+import path from 'path';
+import {
+  declare,
+  deploy,
+  DeployResponse,
+  ensureTestnetContactable,
+  invoke,
+} from '../testnetInterface';
+import * as fs from 'fs';
+
+const testPath = `${path.resolve(__dirname, '..')}/interface_call_forwarder`;
```

Comment:
path.resolve(__dirname, '..', 'interface_call_forwarder');
path.resolve(__dirname, '..', 'interface_call_forwarder', 'contract.cairo');
similar refactoring for all these